doc/man3/BIO_set_callback.pod

```diff
@@ -111,39 +111,39 @@ or
 
 is called before the free operation.
 
-=item B<BIO_read_ex(b, out, outl, read)>
+=item B<BIO_read_ex(b, data, dlen, read)>
```

Comment:
Looks like I missed that part of the change, as the implementation uses readbytes.